index.html

```diff
@@ -48,78 +57,127 @@ <h2>Tonic</h2>
                 </ul>
+                <p class="mobileP">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <p class="deskP">A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
+                <div class="mobileTeck">
+                    <ul class="teck-list">
+                        <li><button class="mobile-btn" type="submit">HTML</button></li>
+                        <li><button class="mobile-btn" type="submit">CSS</button></li>
+                        <li><button class="mobile-btn" type="submit">JavaScript</button></li>
+                    </ul>
+                </div>
+                <div class="deskTeck">
+                    <ul class="teck-list">
+                        <li><button class="mobile-btn" type="submit">HTML</button></li>
+                        <li><button class="mobile-btn" type="submit">Ruby on Rails</button></li>
+                        <li><button class="mobile-btn" type="submit">CSS</button></li>
+                        <li><button class="mobile-btn" type="submit">JavaScript</button></li>
+                    </ul>
+                </div>
+                <button class="see-mobile-btn" type="submit">See Projects</button>
             </div>
-            <button class="see-mobile-btn" type="submit">See Projects</button>
         </section>
     </section>
     <!-- about area -->
     <section class="about">
-        <h5>About Myself</h5>
-        <p class="about-p">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-        <span>LET'S CONNECT</span>
-        <ul class="media-links">
-            <li><a href="#"><i class="fa fa-twitter"></i></a></li>
-            <li><a href="#"><i class="fa-brands fa-linkedin-in"></i></a></li>
-            <li><a href="#"><i class="fa-brands fa-m"></i></a></li>
-            <li><a href="#"><i class="fa-brands fa-github"></i></a></li>
-            <li><a href="#"><i class="fa-brands fa-angellist"></i></a></li> 
-        </ul>
-        <button class="resume-mobile-btn" type="submit">Get my resume</button>
+        <div class="about-grid-column">
+            <h5>About Myself</h5>
+            <p class="about-p">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <span class="connect">LET'S CONNECT</span>
+            <ul class="media-links" id="connect-media-links">
+                <li><a href="#"><i class="fa fa-twitter"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-linkedin-in"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-m"></i></a></li>
```

Comment:
- [ ] please, the arrow are almost at the end of the line. kindly implement that
- [ ] please use the correct text color for the HTML, css and js text below the square icons
- [ ] kindly fix your `see my resume` button so it follows the Figma design.
- [ ] please the about section has a border radius on the top right. kindly implement that in your design as well.
yours

![Screenshot (1238)_LI](https://github.com/efmomoh/Portfolio-setup-and-mobile-first/assets/65298048/7f703301-8f48-4896-9cb1-88cc8ea5d03f)
figma
![Screenshot (1238)](https://github.com/efmomoh/Portfolio-setup-and-mobile-first/assets/65298048/fccc1b06-29ec-4f4b-8253-0d56ec652b96)